src/app/core/auth/auth.service.ts

```diff
@@ -156,6 +156,15 @@ export class AuthService {
     return this.store.pipe(select(isAuthenticatedLoaded));
   }
 
+  public setExternalAuthStatus(external: boolean) {
```

Comment:
Could we add TypeDocs to this new method to explain the purpose?  It appears this should be set to `true` whenever an authentication cookie exists (as that implies an external authentication system like Shib is being used).. as is done in `auth.effects.ts`